autonomy/cli/utils/spec_helpers.py

```diff
@@ -158,11 +153,7 @@ def check_all(
             ),
         ]
     )
-    # TODO: fix this implementation:
-    # - above file names should be constant and explained in helper text
-    # - investigate: either a) we need to do this in the order of the inverse dependency tree, so that
-    # dependencies are already loaded. Otherwise dependencies might not be present. Or b)
-    # we need to ensure relative modules are loaded too (https://docs.python.org/3/library/importlib.html#importlib.import_module)?
```

Comment:
We actually don't need to do anything extra, it looks like `import_module` takes care of dependencies